client/src/components/CompletedProfile/CompletedProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import styled from "styled-components";
+import { theme, mq } from "constants/theme";
+import BaseButton from "components/Button/BaseButton";
+
+const { tablet } = mq;
+
+// ICONS
+const { colors, typography } = theme;
+
+export const ProfileCompletedWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  background-color: #eeeeef;
+  height: 100%;
+`;
+
+export const ProfileCompletedHeader = styled.div`
+  height: 40vh;
+  width: 100%;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  background-color: ${colors.royalBlue};
+  border-bottom-right-radius: 15px;
+
+  @media screen and (min-width: ${tablet.wide.minWidth}) {
+    background: 0 0;
+  }
+`;
+
+export const ProfileCompletedButtonsWrapper = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  position: relative;
+  top: -3rem;
+  width: 100%;
+  padding: 0 1em;
```

Comment:
is there any reason you are using here 'em' instead of 'rem'?

Reply:
My bad, will make the change